.github/CONTRIBUTING.md

```diff
@@ -212,7 +212,27 @@ invoke `make format`.
 The target `make formatall` in the root will automatically run all style checks
 and make any required changes. PRs must pass the formatting checks before landing.
 
-### Step 8: Landing
+### Step 8: Licensing
+
+Tock is dual-licensed under Apache2/MIT. Files in Tock require a short header
+indicating this. In 99% of cases, the following text should be at the top of every file:
+
+> © Copyright 2020[-20XX] The Tock Project Authors.
```

Comment:
As I mentioned in the call and speaking as not a lawyer. I don't see how you can do this per file without a CLA. Other projects without CLAs (Linux, QEMU, etc.) have this Copyrighted to major contributors.

Reply:
Agreed. Rust slowly purged copyright notices from all of their projects. I think Cargo's PR sums up the rationale the best: https://github.com/rust-lang/cargo/pull/4331

I've updated this to remove copyright language from the header template.